src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -127,6 +173,8 @@ IOUAmountPage.displayName = 'IOUAmountPage';
 IOUAmountPage.propTypes = propTypes;
 IOUAmountPage.defaultProps = defaultProps;
 
-export default withWindowDimensions(withOnyx({
-    iou: {key: ONYXKEYS.IOU},
-})(IOUAmountPage));
+export default withWindowDimensions(
+    withOnyx({
+        iou: {key: ONYXKEYS.IOU},
+    })(IOUAmountPage),
+);
```

Comment:
Let's use `compose` to export components that use multiple higher-order-components.
```suggestion
export default compose(
    withWindowDimensions,
    withOnyx({
        iou: {key: ONYXKEYS.IOU},
    }),
)(IOUAmountPage);
```